jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/RuntimeGroup.kt

```diff
@@ -43,6 +43,7 @@ enum class RuntimeGroup {
     fun determineRuntime(module: Module): Runtime? = info.asSequence().mapNotNull { it.determineRuntime(module) }.firstOrNull()
     fun getModuleType(): ModuleType<*>? = info.asSequence().mapNotNull { it.getModuleType() }.firstOrNull()
     fun getIdeSdkType(): SdkType? = info.asSequence().mapNotNull { it.getIdeSdkType() }.firstOrNull()
+    fun supportSamBuild(): Boolean = info.asSequence().mapNotNull { it.supportSamBuild() }.first()
```

Comment:
Looking at the lines above this one, I don't know if `.first()` is safe, in that `.mapNotNull` or `.asSequence` could produce an empty list

Reply:
Updated the logic to ask for `all` support sam build.